routes/attachments.js

```diff
@@ -7,22 +7,22 @@ const router = express.Router()
 const ResultCoordinates = require('../lib/resultCoordinates')
 
 // Get a proposed patch for a specific revision of a component
-router.get('/:contentToken', asyncMiddleware(getContent))
+router.get('/:token', asyncMiddleware(getAttachment))
 
-async function getContent(request, response) {
+async function getAttachment(request, response) {
   // TODO this is a hack. The coordinates are not intended to be sparse. Here make sure the
   // token is the "namespace" to avoid the '-' showing up. Fix is likely to have an explicit
-  // ContentCoordinates class that has the right shape and generalize the stores to just take
+  // AttachmentCoordinates class that has the right shape and generalize the stores to just take
   // "coordinates" and not worry about how they are structured.
-  const coordinates = new ResultCoordinates('content', null, request.params.contentToken)
-  const result = await contentStore.get(coordinates)
-  if (result) return response.status(200).send(result.content)
+  const coordinates = new ResultCoordinates('attachment', null, request.params.token)
+  const result = await harvestStore.get(coordinates)
+  if (result) return response.status(200).send(result.attachment)
```

Comment:
Small style question, I normally try and keep the happy path on the "root" of the method and have the guard causes bail out (because often there's more bail out conditions then success conditions), so I would have written this as:

```javascript
if (!result) return response.sendStatus(404)
return response.status(200).send(result.attachment)
```